### ISSUE TITLE
Auto-scroll not working for drag-drop in DetailsList in IE

### ISSUE THREAD

**Issue by arkogupta** [OfficeDev/office-ui-fabric-react#7482]
### Environment Information 
 
 

- __Browser and OS versions__: IE 11 , Windows 10, server 2012R2
### Please provide a reproduction of the bug in a codepen:
 

Video link which shows a repro of the bug : 

 [https://microsoft-my.sharepoint.com/:V:/p/hatrived/EVWjnKwFQwhNuttthPfeGYgBN_1xPuqxA8MMQGeb7cAoBg?e=4%3apOmTNj%3aorigemail&at=9](url)
#### Actual behavior:
 
The list is **not** scrolling horizontally automatically when the dragged column header reaches the end of the list.

#### Expected behavior:
 
The list should scroll horizontally automatically when the dragged column header reaches the end of the list.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? 
 
Requested priority: (Blocking, High, Normal, Low) High
 
Products/sites affected: (if applicable) SharePoint

**Comment by arkogupta** [comment]
@micahgodbolt @KevinTCoughlin Could anyone please help out with this ?

Thanks.

**Comment by dzearing** [comment]
@shagra-ms just a heads up here; probably we want to add `AutoScroll` for draggable columns.

**Comment by shagra-ms** [comment]
Thanks for looping me in @dzearing. We have picked up this item.

It is interesting to note that with HTML 5 Drag-Drop support - all browsers already support autoscroll. In fact I can drag any image and the IE window does auto scroll in any direction (just to prove that no code is required to enable this). I remember we tested this when we first worked on enabling this scenario. _So this may be a regression_. 

Gaurav will work on this and will try to root cause the issue. May be for consistency we may have to think about integrating `AutoScroll` here.
